tests/fuzzing/README.md

```diff
@@ -1,3 +1,67 @@
 All fuzzers here are run continously through OSS-fuzz.
 
 Link to OSS-fuzz integration: Pending
+
```

Comment:
@sthibaul The comment above says "All fuzzers here are run continously through OSS-fuzz". Is that true? Aren't your fuzzers a different thing?

Reply:
OK got it. And what about the claim that both are run through OSS-Fuzz?

Reply:
That's just out of date. I have now fixed it.

Reply:
Marvelous, thank you.